task/main.py

```diff
@@ -41,33 +44,34 @@ def __init__(self, event):
         table_resource = boto3.resource('dynamodb', region_name='us-west-2')
         self.db_table = table_resource.Table(os.getenv('table_name', default='DiscoverGranulesLock'))
         # Note: logging.DEBUG and logging.INFO seem to have the same result with the cumulus debugger
-        logging_level = logging.INFO if os.getenv('enable_logging', 'false').lower() == 'true' else logging.WARNING
-        self.logger = CumulusLogger(name='Recursive-Discover-Granules', level=logging_level)
 
     def discover(self):
         """
         Helper function to kick off the entire discover process
         """
         granules = self.collection.get('meta', {}).get('granules', None)
         if granules:
-            self.logger.info(f'Received {len(granules)} to reingest.')
+            logger.info(f'Received {len(granules)} to reingest.')
             granule_dict = {}
             for granule in granules:
                 self.populate_dict(granule_dict, key=granule, etag=None, last_mod=None)
             output = self.cumulus_output_generator(granule_dict)
             pass
         else:
             granule_dict = self.discover_granules()
-            self.logger.info('##########')
-            self.logger.info(f'Discovered {len(granule_dict)} granules.')
+            if not granule_dict:
+                logger.warning(f'Warning: Found 0 granules at the provided location.')
+            else:
+                logger.info(f'Discovered {len(granule_dict)} granules for update processing.')
             self.check_granule_updates_db(granule_dict)
 
-            self.logger.info(f'{len(granule_dict)} granules for ingest.')
             output = self.cumulus_output_generator(granule_dict)
+            print(f'Returning cumulus output for {len(output)} {self.collection.get("name")} granules.')
```

Comment:
@amarouane-ABDELHAK The collection name is printed after the cumulus output is generated. I can add above as well if needed?

Reply:
Yes please add it to see how many are discovered (or none discovered)